index.html

```diff
@@ -63,6 +63,224 @@ <h1>I’m Alexandra. Glad to see you!</h1>
         </section>
 
     </div>
+    <!-- MY Works section -->
+    <section class="works-container">
+        <!-- Works section items -->
+
+        <div class="works-grid-item">
+            <div class="template-img flex-item-width">
+                <img src="img/Portfolio.png" alt="">
+            </div>
+            <div class="Tonic flex-item-width">
+                <h2>Tonic</h2>
+            </div>
+            <div class="ol-list flex-item-width">
+                <ol>
+                    <li id="canopy">CANOPY</li>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
```

Comment:
- [ ] You really put in an effort based on the previous reviewer's suggestions 👏 . However, you still have a little change to make regarding the alignment of your pictures and heading. This is emphasised to make sure we have the best user experience in mind, and sticking to requirements 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/235325825-5b55aef4-5313-43d6-9ef1-5c41fffc0b76.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/235325862-4886d61a-1926-4131-9ab0-ae332f99a316.png)